smaps/src/main/webapp/index.html

```diff
@@ -0,0 +1,66 @@
+<!--
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link href='/css/style.css' rel='stylesheet' type='text/css'>
+    <title>Smaps Visualizer</title>
+</head>
+
+<body class="roboto" style="margin: 2em">
+    <div class="container">
+        <div class="row justify-content-md-center" style="height: 50px">
+            <div id="smaps-title">
+                <span class="gblue">s</span>
+                <span class="gred">m</span>
+                <span class="gyellow">a</span>
+                <span class="gblue">p</span>
+                <span class="ggreen">s</span>
+            </div>
+        </div>
+        <div class="row justify-content-md-center">
+            <p id="visualizer-title" class="ggray">Visualizer</p>
+        </div>
+        <div class="row justify-content-md-center">
+            <p>This application takes a process' smap dump and creates useful charts/visualizations from it.</p>
```

Comment:
Let's stick to referring to it as "smaps" (instead of "smap").
I think here too, we could drop the word dump and just say "takes a process' smaps and creates..."